native-sql-engine/core/src/main/scala/com/intel/oap/expression/ColumnarExpressionConverter.scala

```diff
@@ -271,11 +271,7 @@ object ColumnarExpressionConverter extends Logging {
             columnarDivide,
             expr)
         }
-      case oaps: com.intel.oap.expression.ColumnarScalarSubquery =>
```

Comment:
ColumnarScalarSubquery is useless, so we removed the code where it is used. Not relevant to spark 3.1/3.2 compatibility.